packages/util/src/assignment.ts

```diff
@@ -34,9 +87,10 @@ export function assignmentFn(participantIndex: ParticipantIndexType, assignmentP
  * @param meetupCount
  */
 export function meetupIndex(participantIndex: ParticipantIndexType, assignmentParams: AssignmentParams, meetupCount: MeetupIndexType): MeetupIndexType {
-    const result = assignmentFn(participantIndex, assignmentParams, meetupCount).add(new BN(1));
 
-    return participantIndex.registry.createTypeUnsafe('MeetupIndexType', [result]);
+    const result = assignmentFn(participantIndex, assignmentParams, meetupCount);
+
+    return result.addn(1) as MeetupIndexType;
 }
```

Comment:
Quite practical; After `addn` the value is a `BN`, but as it has still the `registry` attached to it, we can simply type-cast it back to `MeetupIndexType` instead of having to call `createType`.